content/akeneo-event-platform/concepts.md

```diff
@@ -1,5 +1,128 @@
+
+The following properties represent a subscription:
+
+| Property | Value | Description |
+| --- | --- | --- |
+| `id` | Automatically populated | Identifier of the subscription within the Akeneo Event Platform |
+| `source` | Populated by the user at creation | Source of the event (currently, the only source available is `pim`) |
+| `subject` | From `X-PIM-URL` header parameter | URL of the targeted source |
+| `type` | Populated by the user at creation | Type of the subscription (currently, there are two available types:  `https` and `pubsub`) |
+| `events` | Populated by the user at creation | A list of events that the subscription is tracking |
+| `status` | Automatically populated | The subscription status |
+| `config` | Populated by the user at creation | The subscription configuration, based on the subscription type. See below for further details. |
+
+The statuses for a subscription are:
+
+| Status | Description |
+| --- | --- |
+| `active` | The destination will receive notifications for events monitored by the subscription |
+| `deleted` | The subscription is inactive and cannot be reactivated |
+| `suspended` | The subscription has been deactivated by the event platform itself due to excessive errors, or manually by the user. However, you can resume it. Suspending a subscription stops all events from being sent to it. Events are not saved and are lost until the subscription is resumed |
+| `revoked` | The subscription has been automatically deactivated because the destination responded with an HTTP 404. You can manually reactivate it. Suspending a subscription halts all event deliveries to this subscription. Events are not saved and are lost until the subscription is resumed |
+
+### Best practices
+
+#### HTTP subscription
+
+##### Configuration
+
+For the `https` type, the `config` property requires:
+
+- a URL using the HTTPS protocol;
+- a URL that accepts HEAD requests and responds with an HTTP status code other than 404 or 5xx.
+
+Additionally, it requires at least a primary secret (with an optional secondary secret) to sign the messages sent to the specified URL.
+
+```json
+
+"url": "https://your_webhook_url",
+"secret": {
+    "primary": "your_primary_secret_to_sign_the_payload",
+    "secondary": "your_secondary_secret_to_sign_the_payload"
+}
+```
+
+##### HMAC signature for webhook payloads
+
+HMAC (Hash-based Message Authentication Code) is a method used to verify the integrity and authenticity of a message. Using a secret key shared between the sender and receiver, HMAC ensures that the payload is untampered and originates from a trusted source.
+
+For HTTPS subscriptions, we sign each payload using the secret you provided in the configuration.
+
+We include the signature in the payload headers:
+
+- `X-AKENEO-SIGNATURE-PRIMARY`: Contains the signature using your provided primary secret.
```

Comment:
- `X-AKENEO-SIGNATURE-PRIMARY`: Contains the signature using the primary secret you provided.